js/app/src/Login.svelte

```diff
@@ -37,7 +37,7 @@
 	<Column variant="panel" min_width={480}>
 		<h2>{$_("login.login")}</h2>
 		{#if auth_message}
-			<p class="auth">{auth_message}</p>
+			<p class="auth">{@html auth_message}</p>
```

Comment:
I see, cool thanks.  I just tried to remove the whole class, I don't know this stuff enough.

Reply:
No problem thanks for flagging this!